images/ubuntu-slim/Dockerfile.build

```diff
@@ -21,23 +21,18 @@ RUN echo "Yes, do as I say!" | apt-get purge \
     libudev1 \
     tzdata
 
-RUN apt-get autoremove -y
-
+# replace shell bash with sh
 RUN sed -i 's/bash/sh/g' /etc/passwd
 
-WORKDIR /usr/share
```

Comment:
Since this is removed the `rm -rf` section needs to be updated to reference `/usr/share/common-licenses` instead of `common-licenses`